life_hc/MySQL/WantedSystem/fn_wantedBounty.sqf

```diff
@@ -18,7 +18,7 @@ params [
     ["_half",false,[false]]
 ];
 
-if (isNull _civ || {isNull _cop}) exitWith {};
+if (isNull _civ || isNull _cop) exitWith {};
```

Comment:
I believe this was intended. See https://community.bistudio.com/wiki/Code_Optimisation#Lazy_evaluation